### PR TITLE
Add reset LogUrl and reset Appcenter secret UI

### PR DESCRIPTION
**Pull request by Cloudwu1**
## Description

1. Launch Unity test demo app, 
2. Check if there is a "Reset" button for Log Url and App secret on App Center tab.

image

## Related PRs or issues

[AB#67487](https://dev.azure.com/msmobilecenter/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/67487)

### CODE REVIEW

**Review by annakocheshkova — DISMISSED** [review]

**Review by AnastasiaKubova — DISMISSED** [review]

**Review by annakocheshkova — DISMISSED** [review]

File: Assets/Puppet/PuppetAppCenter.cs [hunk 1]
```diff
@@ -254,6 +254,28 @@ public void SetLogUrl(string logUrl)
...
+    }
+
+    public void ClearAppSecret()
+    {
+        AppSecret.text = "";
+        AppSecretCached = null;
+        PlayerPrefs.DeleteKey(AppSecretKey);
+        AppCenter.ParseAndSaveSecretForPlatform(null);
+#if UNITY_ANDROID && !UNITY_EDITOR
+        AndroidUtility.SetPreferenceString(AppSecretAndroidKey, null);
```
Comment (guperrot) [inline Assets/Puppet/PuppetAppCenter.cs:275]: How does it work on iOS then?

Comment (annakocheshkova) [reply Assets/Puppet/PuppetAppCenter.cs:275]: Via `PlayerPrefs`. The workaround is needed only for Android.

Comment (MatkovIvan) [reply Assets/Puppet/PuppetAppCenter.cs:275]: Let's move `PlayerPrefs` stuff to `#if !UNITY_ANDROID` then

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

File: Assets/Puppet/PuppetAppCenter.cs (hunk 1 above)
Comment (annakocheshkova) [reply Assets/Puppet/PuppetAppCenter.cs:275]: It works this way. Let's leave a comment in code.

Comment (jaeklim) [reply Assets/Puppet/PuppetAppCenter.cs:275]: @annakocheshkova Can you explain why we need a workaround only for Android?

File: AppCenterDemoApp/Assets/Puppet/PuppetAppCenter.cs [hunk 2]
```diff
@@ -223,6 +223,7 @@ public void SetCustomDialog(bool enable)
 
     public void SetStartupMode(int startupMode)
     {
+        // A workaround for android. Both calls to PlayerPrefs and AndroidUtility are needed
```
Comment (guperrot) [inline AppCenterDemoApp/Assets/Puppet/PuppetAppCenter.cs]: Please explain why though (and end comments with dots).

**Review by guperrot — CHANGES_REQUESTED** [review]

File: Assets/Puppet/PuppetAppCenter.cs (hunk 1 above)
Comment (annakocheshkova) [reply Assets/Puppet/PuppetAppCenter.cs:275]: Done

**Review by guperrot — APPROVED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**